src/mqt/bench/benchmarks/bmw_quark_copula.py

```diff
@@ -37,41 +36,55 @@ def create_circuit(num_qubits: int, depth: int = 2) -> QuantumCircuit:
         num_qubits: number of qubits of the returned quantum circuit
         depth: depth of the returned quantum circuit
     """
-    rng = np.random.default_rng(10)
+    assert num_qubits % 2 == 0, "Number of qubits must be divisible by 2."
 
     n_registers = 2
     n = num_qubits // n_registers
-
     qc = QuantumCircuit(num_qubits)
 
-    for k in range(n):
-        qc.h(k)
+    # === Compute number of parameters ===
+    num_single_qubit_gates = depth * n_registers * n * 3
+    num_rxx_gates = depth * n_registers * comb(n, 2)
+    total_params = num_single_qubit_gates + num_rxx_gates
+
+    param_vector = ParameterVector("p", total_params)
+
+    param_index = 0
+
+    def get_param() -> float | Parameter:
```

Comment:
The return value of this function will never be float.

Reply:
Fixed it.